.github/workflows/lint-codeql.yaml

```diff
@@ -3,12 +3,20 @@ name: codeql
 on:
   push:
     branches:
+    - master
     - v1.10
     - v1.9
     - v1.8
+    paths:
+      - '*.go'
```

Comment:
For reviewers: Are there other files that should be included here? Configuration files for CodeQL I've missed maybe?

Reply:
Maybe add `go.mod` and `go.sum` for completeness. CodeQL itself doesn't have any config in the repo (besides the workflow file that triggers it).

Reply:
Done.